_rules/sequentially-focusable-element-has-visible-focus-through-styling-f4e323.md

```diff
@@ -0,0 +1,411 @@
+---
+id: f4e323
+name: Element in sequential focus order has visible focus though styling
```

Comment:
Note to reviewers:
I initially wrote this rule that is too strict for WCAG (2.4.7 is notably weak in its requirements). After discussing with @WilcoFiers, we decided to go for the "minimal version" (which is the other rule).

I was reluctant about throwing away all the work here 😉 So I decided to try and keep it as a "best practice" rule (it doesn't really check for 2.4.11 either). Let me know what you think about it and whether we should keep it or not…
(if we keep it, we'll probably want to move it to another PR for separation of concerns, of course, I'll do that if initial feedback on it is good 😀)

Reply:
I've just browsed this rule, not really reviewed it (because, as you say, we should move it to a different PR if we decide to keep it).

I feel your pain, because of all the effort you must have put into this, but I'm not really sure it will add much value to the other visible focus rule.